file/lib/src/sentry_io_overrides.dart

```diff
@@ -0,0 +1,23 @@
+
+import 'dart:io';
+import 'package:sentry/sentry.dart';
+
+import 'sentry_file.dart';
+
+class SentryIoOverrides extends IOOverrides {
+  final Hub _hub;
+  final SentryOptions _options;
+
+  SentryIoOverrides(this._hub, this._options);
+
+  @override
+  File createFile(String path) {
+    if (_options.platformChecker.isWeb || !_options.isTracingEnabled()) {
+      return super.createFile(path);
+    }
```

Comment:
`_options.platformChecker.isWeb` is also by definition always false, since the outer code can't be executed on web platforms, since it relies on `dart:io`